script/openqa-bootstrap

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -x
+
+
+# add extra repos for leap
+source /etc/os-release
+if $NAME == "openSUSE Leap" ; then
+	zypper -n addrepo obs://devel:openQA devel:openQA
+	zypper -n addrepo obs://devel:openQA:Leap:${VERSION} devel:openQA:Leap:${VERSION}
+	zypper -n  --gpg-auto-import-keys refresh
+fi
+
+
+# install packages
+zypper -n install --no-recommends openQA-local-db apache2 openQA-worker qemu-kvm sudo
+
+
+# setup database
+systemctl enable --now postgresql
+su postgres -c "createuser -D geekotest"
+su postgres -c "createdb -O geekotest openqa"
```

Comment:
Can you rely on the openqa-setup-db systemd service here to prevent duplication?

Reply:
No, because consider it as very ugly to use a systemd job for a task, that is only executed once.
This is the job of an sh script - not of a systemd.service file.
Also it will help users to understand, what this script does and therefore act as some kind of setup documentation.

Reply:
Well, we should be consistent within this repo still. We can suggest to replace the existing openqa-setup-db service with a script instead as well